app/code/Magento/Sales/Block/Adminhtml/Order/View/Tab/Shipments.php

```diff
@@ -21,19 +23,27 @@ class Shipments extends \Magento\Framework\View\Element\Text\ListText implements
      */
     protected $_coreRegistry = null;
 
+    /**
+     * @var AuthorizationInterface
+     */
+    private $authorization;
+    
     /**
      * Collection factory
      *
      * @param \Magento\Framework\View\Element\Context $context
      * @param \Magento\Framework\Registry $coreRegistry
+     * @param AuthorizationInterface $authorization
      * @param array $data
      */
     public function __construct(
         \Magento\Framework\View\Element\Context $context,
         \Magento\Framework\Registry $coreRegistry,
+        AuthorizationInterface $authorization,
```

Comment:
This class is marked with @api class, so we can't introduce any backward incompatible changes.

Could you make this argument the last one, make it optional with fallback to retrieving this object from the object manager?

Reply:
Done